2020-01-31 01:41:56 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 29
2020-01-31 01:44:14 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 29
2020-01-31 01:45:42 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 26
2020-01-31 01:50:49 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 26
2020-01-31 01:51:04 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 26
2020-01-31 01:51:56 - ERROR - DriverNotFound in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 26
2020-01-31 01:54:28 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 26
2020-01-31 01:55:27 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 26
2020-01-31 01:55:53 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 9
2020-01-31 01:55:57 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 9
2020-01-31 01:57:11 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 9
2020-01-31 01:57:15 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 26
2020-01-31 01:57:19 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 26
2020-01-31 02:00:38 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 26
2020-01-31 02:03:23 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 10
2020-01-31 02:04:34 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 22
2020-01-31 02:09:02 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 26
2020-01-31 02:17:12 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-01-31 02:17:48 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 28
2020-01-31 02:18:44 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 32
2020-01-31 02:21:05 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 32
2020-01-31 02:23:26 - ERROR - KeyError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 32
2020-01-31 02:24:17 - ERROR - InsecureTransportError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 32
2020-02-01 12:03:22 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 29
2020-02-01 12:04:02 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 26
2020-02-01 12:34:25 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 26
2020-02-01 12:57:51 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 13:23:21 - ERROR - InsecureTransportError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 33
2020-02-01 13:25:52 - ERROR - InsecureTransportError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 33
2020-02-01 13:28:57 - ERROR - InsecureTransportError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 33
2020-02-01 13:31:50 - ERROR - InsecureTransportError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 32
2020-02-01 13:32:43 - ERROR - CustomOAuth2Error in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 32
2020-02-01 13:37:47 - ERROR - CustomOAuth2Error in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 32
2020-02-01 13:37:52 - ERROR - CustomOAuth2Error in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 32
2020-02-01 13:41:34 - ERROR - CustomOAuth2Error in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 32
2020-02-01 13:42:03 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 32
2020-02-01 13:43:06 - ERROR - KeyError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 32
2020-02-01 13:43:09 - ERROR - KeyError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 32
2020-02-01 13:43:34 - ERROR - CustomOAuth2Error in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 33
2020-02-01 13:43:39 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 33
2020-02-01 13:45:12 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 32
2020-02-01 13:46:33 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 32
2020-02-01 13:54:00 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 33
2020-02-01 13:54:14 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 33
2020-02-01 13:54:19 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 33
2020-02-01 13:56:11 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 33
2020-02-01 13:57:57 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 33
2020-02-01 13:58:31 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 34
2020-02-01 13:59:43 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 34
2020-02-01 14:00:12 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 30
2020-02-01 14:00:36 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 30
2020-02-01 14:00:50 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 30
2020-02-01 14:03:55 - ERROR - ValueError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 31
2020-02-01 14:04:41 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 33
2020-02-01 14:18:13 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 33
2020-02-01 14:36:44 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 33
2020-02-01 14:36:49 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 33
2020-02-01 14:44:15 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 28
2020-02-01 15:03:21 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 27
2020-02-01 15:09:01 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 31
2020-02-01 15:12:26 - ERROR - JSONDecodeError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 27
2020-02-01 15:12:30 - ERROR - JSONDecodeError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 27
2020-02-01 15:13:42 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 27
2020-02-01 15:14:26 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 27
2020-02-01 15:14:46 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 27
2020-02-01 15:23:09 - ERROR - CustomOAuth2Error in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 28
2020-02-01 15:23:14 - ERROR - CustomOAuth2Error in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 28
2020-02-01 15:25:11 - ERROR - CustomOAuth2Error in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 29
2020-02-01 15:26:42 - ERROR - MismatchingStateError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 30
2020-02-01 15:28:43 - ERROR - MismatchingStateError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 30
2020-02-01 15:30:19 - ERROR - MismatchingStateError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 31
2020-02-01 15:32:08 - ERROR - CustomOAuth2Error in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 28
2020-02-01 15:32:29 - ERROR - CustomOAuth2Error in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 28
2020-02-01 15:32:34 - ERROR - CustomOAuth2Error in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 28
2020-02-01 15:33:56 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 30
2020-02-01 15:34:03 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 30
2020-02-01 15:36:26 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 28
2020-02-01 15:41:38 - ERROR - MissingTokenError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 28
2020-02-01 15:43:46 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 27
2020-02-01 15:43:50 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteFacebookDriver.py on line 27
2020-02-01 16:02:45 - ERROR - InvalidGrantError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteGoogleDriver.py on line 32
2020-02-01 16:03:56 - ERROR - InvalidGrantError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteGoogleDriver.py on line 32
2020-02-01 16:04:01 - ERROR - InvalidGrantError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteGoogleDriver.py on line 32
2020-02-01 16:07:40 - ERROR - InvalidGrantError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteGoogleDriver.py on line 32
2020-02-01 16:12:30 - ERROR - InvalidGrantError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteGoogleDriver.py on line 32
2020-02-01 16:12:34 - ERROR - InvalidGrantError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteGoogleDriver.py on line 32
2020-02-01 18:25:12 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/helpers.py on line 27
2020-02-01 18:25:17 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/helpers.py on line 27
2020-02-01 18:29:15 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/helpers.py on line 15
2020-02-01 18:32:13 - ERROR - ValueError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/helpers.py on line 24
2020-02-01 18:34:21 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/strategy/MasoniteStrategy.py on line 24
2020-02-01 18:36:31 - ERROR - IndexError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/helpers.py on line 16
2020-02-01 18:37:33 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 18:39:42 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 18:41:09 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 18:42:13 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 18:43:50 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 18:44:10 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 18:44:31 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 18:52:37 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 18:56:30 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 18:56:47 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 18:58:55 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 18:59:10 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 19:14:54 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/AbstractDriver.py on line 28
2020-02-01 19:16:44 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 19:18:38 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 19:22:10 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 19:25:13 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 19:26:44 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 19:27:23 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 19:29:19 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 19:36:22 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 19:40:25 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 19:52:15 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 19:53:08 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 19:57:08 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 19:57:15 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 19:57:29 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 19:57:43 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 19:58:00 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 19:58:40 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 20:00:48 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 20:02:09 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 20:02:44 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 20:04:34 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 20:06:16 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 20:08:19 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 20:09:48 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 20:10:40 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 20:11:00 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 20:11:16 - ERROR - ResponseError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 20:12:20 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 20:13:49 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 20:14:31 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 20:14:52 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 20:15:46 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 20:16:18 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 20:17:04 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 20:17:20 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 20:17:32 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 20:18:32 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 20:18:48 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 20:19:44 - ERROR - ContainerError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 20:23:31 - ERROR - ResponseError in /home/alcoy/PycharmProjects/masonite_socialite/bootstrap/start.py on line 38
2020-02-01 20:46:09 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 33
2020-02-01 20:46:13 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 33
2020-02-01 21:59:03 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/strategy/MasoniteStrategy.py on line 73
2020-02-01 21:59:08 - ERROR - HTTPError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/actions.py on line 7
2020-02-01 22:05:38 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/strategy/MasoniteStrategy.py on line 73
2020-02-01 22:05:43 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/strategy/MasoniteStrategy.py on line 73
2020-02-01 22:08:54 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/strategy/MasoniteStrategy.py on line 74
2020-02-01 22:27:22 - ERROR - HTTPError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/actions.py on line 30
2020-02-01 22:41:57 - ERROR - DriverNotFound in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 28
2020-02-01 22:42:15 - ERROR - DriverNotFound in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 28
2020-02-01 22:55:20 - ERROR - DriverNotFound in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 28
2020-02-01 23:01:58 - ERROR - DriverNotFound in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 28
2020-02-01 23:04:15 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 28
2020-02-01 23:06:46 - ERROR - DriverNotFound in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/SocialiteManager.py on line 9
2020-02-01 23:07:12 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 28
2020-02-01 23:09:13 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 28
2020-02-01 23:10:04 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 28
2020-02-01 23:13:44 - ERROR - AttributeError in /home/alcoy/PycharmProjects/masonite_socialite/app/http/controllers/WelcomeController.py on line 28
2020-02-01 23:21:03 - ERROR - HTTPError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/actions.py on line 30
2020-02-01 23:41:41 - ERROR - MissingBackend in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/helpers.py on line 24
2020-02-01 23:42:04 - ERROR - MissingBackend in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/helpers.py on line 24
2020-02-01 23:44:45 - ERROR - MissingBackend in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/helpers.py on line 24
2020-02-01 23:44:56 - ERROR - HTTPError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteAuthDriver.py on line 16
2020-02-01 23:49:51 - ERROR - KeyError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteAuthDriver.py on line 21
2020-02-01 23:52:14 - ERROR - AuthCanceled in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/actions.py on line 30
2020-02-01 23:52:36 - ERROR - KeyError in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/drivers/SocialiteAuthDriver.py on line 22
2020-02-01 23:58:15 - ERROR - MissingBackend in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/helpers.py on line 24
2020-02-01 23:59:36 - ERROR - MissingBackend in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/helpers.py on line 24
2020-02-01 23:59:56 - ERROR - MissingBackend in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/helpers.py on line 24
2020-02-01 23:59:59 - ERROR - MissingBackend in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/helpers.py on line 24
2020-02-02 00:00:44 - ERROR - MissingBackend in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/helpers.py on line 24
2020-02-02 00:03:18 - ERROR - MissingBackend in /home/alcoy/PycharmProjects/masonite_socialite/src/socialite/helpers.py on line 24
